specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json

```diff
@@ -5887,6 +5991,19 @@
           "format": "date-time",
           "readOnly": true
         },
+        "invoiceType" : {
```

Comment:
Technically if this was a stable api version, it would be a breaking change. Please communicate this accordingly.